rd_ui/app/views/users/show.html

```diff
@@ -36,6 +36,10 @@
                         <input name="email" type="email" class="form-control" ng-model="user.email" required/>
                         <input-errors errors="userSettingsForm.email.$error"/>
                     </div>
+                    <div class="form-group required" ng-if="showSettings">
```

Comment:
It should check if the user is admin (`ng-if="currentUser.hasPermission('admin')"`).
Also you can remove the admin checkbox.